ibm/service/power/resource_ibm_pi_shared_processor_pool.go

```diff
@@ -265,6 +283,9 @@ func resourceIBMPISharedProcessorPoolRead(ctx context.Context, d *schema.Resourc
 	}
 
 	d.Set(Arg_CloudInstanceID, cloudInstanceID)
+	if response.SharedProcessorPool.Crn != "" {
+		d.Set(Attr_CRN, response.SharedProcessorPool.Crn)
+	}
```

Comment:
This is duplicated, since you have that group with the tag

Reply:
Fixed